src/app/helpers/formTemplate/formTemplate.ts

```diff
@@ -13,68 +13,65 @@ import { takeUntil } from "rxjs/operators";
 import { PageComponent } from "../page/pageComponent";
 import { PageInfo } from "../page/pageInfo";
 
+export interface FormTemplateOptions<Model> {
+  successMsg: (model: Partial<Model>) => string;
+  failureMsg: (err: ApiErrorDetails) => string;
+  onSuccess?: (_: Model | void) => void;
```

Comment:
it doesn't make much sense to use a discard variable here
```suggestion
  onSuccess?: (model: Model | void) => void;
```